data/trees.yml

```diff
@@ -0,0 +1,65 @@
+---
+# four types of coffee tree strains (variants of coffea arabica) the co-op grew in Guatemala
```

Comment:
this allows us to delete some of what we have now from the old project but is currently unused